src/Uno.UI.Composition/Composition/Visual.cs

```diff
@@ -32,6 +32,18 @@ internal Visual(Compositor compositor) : base(compositor)
 
 		internal VisualInteractionSource? VisualInteractionSource { get; set; }
 
+		internal bool IsTranslationEnabled { get; set; }
+
+		internal Vector3 GetTotalOffset()
+		{
+			if (IsTranslationEnabled && Properties.TryGetVector3("Translation", out var translation) == CompositionGetValueStatus.Succeeded)
```

Comment:
I would suspect that the `"Translation"` should be reflected in the `TransformMatrix`? If so, you should arrange the `TransformableObjectExtensions.GetTransform` extension method instead (BTW I would probably move that method as an extensions method to keep the `Visual` class as simple as possible)

Reply:
@dr1rrb Nope, it looks like it's not reflected in TransformMatrix. It remains identity on WinUI after setting the translation.